packages/expo/cli/register/registerAsync.ts

```diff
@@ -0,0 +1,28 @@
+import openBrowserAsync from 'better-opn';
+
+import { CI } from '../utils/env';
+import { CommandError } from '../utils/errors';
+import { ora } from '../utils/ora';
+
+export async function registerAsync() {
+  const REGISTRATION_URL = `https://expo.dev/signup`;
```

Comment:
Should we use `getExpoWebsiteBaseUrl()` here?

Reply:
That method is used for cases when the URL changes to test staging or dev environments. This URL doesn't really benefit from changing.

Reply:
this command just opens a link, I think the link could be opened manually when testing, it seems over complicated.

Reply:
```js
const REGISTRATION_URL = `${getExpoWebsiteBaseUrl()}/signup`;
```
...or similar, isn't too overly complicated with the benefit of consistency, right? It's not like we need to build an entire thing around it.

Reply:
Importing `getExpoApiBaseUrl` also imports a new file, the file it's in increases the JS bin size by a fair amount.
So we split it out into a new file, but now we have this weird doc block in the file saying we need the file to only contain one function because we use it in the `expo register` command for 'consistency reasons'.